app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_objects = [
+        Person(person["name"], person["age"])
+        for person in people
+    ]
+
+    for person in people:
+        person_object = Person.people[person["name"]]
+
+        if person.get("wife") is not None:
+            wife = Person.people.get(person["wife"])
+            person_object.wife = wife
+        if person.get("husband") is not None:
+            husband = Person.people.get(person["husband"])
```

Comment:
The code assumes that a spouse will always exist in the `Person.people` dictionary. This could result in a `KeyError` if the spouse is not included in the initial list. Consider adding a check to ensure the spouse exists in the dictionary before attempting to access it.